src/styles/blocks/catalog.scss

```diff
@@ -17,6 +21,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    position: relative;
+    transition: all $duration ease;
```

Comment:
specify each prop you apply transition for

Reply:
i do not catch it. what i need to change?